### ISSUE TITLE
Missing output during training on Win 10

### ISSUE THREAD

**Issue by VincentKaras** [DeepSpectrum/DeepSpectrumLite#6]
When training on Win 10, DeepSpectrum Lite exits without error message after start of training process. No training appears to be taking place, and the part of the program which saves and evaluates the model is not reached. Tensorboard logs only Hyperparameters and Graph but no metrics.

System Info: 
OS: Windows 10 
CPU: AMD Ryzen 7 5800X
RAM: 32GB
GPU: Nvidia RTX 2070S

Console Output:

INFO:deepspectrumlite.cli.train:Physical devices:                                                                                                                                                                                                                                         2021-08-23 13:50:32.175008: I tensorflow/compiler/jit/xla_cpu_device.cc:41] Not creating XLA devices, tf_xla_enable_xla_devices not set                                                                                                                                                   2021-08-23 13:50:32.177427: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library nvcuda.dll                                                                                                                                                2021-08-23 13:50:32.216973: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties:                                                                                                                                                                      pciBusID: 0000:2d:00.0 name: NVIDIA GeForce RTX 2070 SUPER computeCapability: 7.5                                                                                                                                                                                                         coreClock: 1.785GHz coreCount: 40 deviceMemorySize: 8.00GiB deviceMemoryBandwidth: 417.29GiB/s                                                                                                                                                                                            2021-08-23 13:50:32.217068: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll                                                                                                                                          2021-08-23 13:50:32.243762: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll                                                                                                                                           2021-08-23 13:50:32.243829: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll                                                                                                                                         2021-08-23 13:50:32.257932: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll                                                                                                                                            2021-08-23 13:50:32.261163: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll                                                                                                                                           2021-08-23 13:50:32.299935: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll                                                                                                                                         2021-08-23 13:50:32.311553: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll                                                                                                                                         2021-08-23 13:50:32.312073: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll                                                                                                                                             2021-08-23 13:50:32.312164: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0                                                                                                                                                                        INFO:deepspectrumlite.cli.train:[PhysicalDevice(name='/physical_device:GPU:0', device_type='GPU')]                                                                                                                                                                                        INFO:deepspectrumlite.cli.train:Loaded hyperparameter configuration.                                                                                                                                                                                                                      INFO:deepspectrumlite.cli.train:Recognised combinations of settings: 1                                                                                                                                                                                                                    2021-08-23 13:50:32.315669: I tensorflow/core/platform/cpu_feature_guard.cc:142] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  AVX2                                     To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.                                                                                                                                                                                               2021-08-23 13:50:32.316491: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1720] Found device 0 with properties:                                                                                                                                                                      pciBusID: 0000:2d:00.0 name: NVIDIA GeForce RTX 2070 SUPER computeCapability: 7.5                                                                                                                                                                                                         coreClock: 1.785GHz coreCount: 40 deviceMemorySize: 8.00GiB deviceMemoryBandwidth: 417.29GiB/s                                                                                                                                                                                            2021-08-23 13:50:32.316560: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudart64_110.dll                                                                                                                                          2021-08-23 13:50:32.316967: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublas64_11.dll                                                                                                                                           2021-08-23 13:50:32.320816: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cublasLt64_11.dll                                                                                                                                         2021-08-23 13:50:32.321255: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cufft64_10.dll                                                                                                                                            2021-08-23 13:50:32.321794: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library curand64_10.dll                                                                                                                                           2021-08-23 13:50:32.322147: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusolver64_10.dll                                                                                                                                         2021-08-23 13:50:32.322682: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cusparse64_11.dll                                                                                                                                         2021-08-23 13:50:32.328163: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll                                                                                                                                             2021-08-23 13:50:32.328682: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1862] Adding visible gpu devices: 0                                                                                                                                                                        2021-08-23 13:50:32.718698: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1261] Device interconnect StreamExecutor with strength 1 edge matrix:                                                                                                                                      2021-08-23 13:50:32.718799: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1267]      0                                                                                                                                                                                               2021-08-23 13:50:32.720523: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1280] 0:   N                                                                                                                                                                                               2021-08-23 13:50:32.721068: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1406] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6611 MB memory) -> physical GPU (device: 0, name: NVIDIA GeForce RTX 2070 SUPER, pci bus id: 0000:2d:00.0, compute capability: 7.5)                                                                                                                                                                                                                                                                               2021-08-23 13:50:32.721856: I tensorflow/compiler/jit/xla_gpu_device.cc:99] Not creating XLA devices, tf_xla_enable_xla_devices not set
INFO:deepspectrumlite.cli.train:Using custom external parser: C:\Users\Vincent\DeepLearning\Projects\Jupyter_Projects\SpeechEmotion\src\label_parser.py:SEWAParser                                                                                                                        INFO:deepspectrumlite.cli.train:--- Starting trial: test_sewa_resnet50_run_0_config_0                                                                                                                                                                                                     INFO:deepspectrumlite.cli.train:{'label_parser': 'C:\\Users\\Vincent\\DeepLearning\\Projects\\Jupyter_Projects\\SpeechEmotion\\src\\label_parser.py:SEWAParser', 'prediction_type': 'regression', 'model_name': 'TransferBaseModel', 'basemodel_name': 'resnet50', 'weights': 'imagenet', 'tb_experiment': 'test_sewa_resnet50', 'tb_run_id': 'test_sewa_resnet50_run_0', 'num_units': 512, 'dropout': 0.2, 'optimizer': 'adam', 'learning_rate': 0.001, 'fine_learning_rate': 0.0001, 'loss': 'mse', 'activation': 'gelu', 'pre_epochs': 20, 'epochs': 100, 'finetune_layer': 0.4, 'batch_size': 16, 'sample_rate': 16000, 'normalize_audio': True, 'chunk_size': 3.0, 'chunk_hop_size': 1.0, 'stft_window_size': 0.128, 'stft_hop_size': 0.064, 'stft_fft_length': 0.128, 'mel_scale': True, 'lower_edge_hertz': 0.0, 'upper_edge_hertz': 8000.0, 'num_mel_bins': 128, 'num_mfccs': 0, 'cep_lifter': 0, 'db_scale': True, 'use_plot_images': True, 'color_map': 'viridis', 'image_width': 224, 'image_height': 224, 'resize_method': 'nearest', 'anti_alias': False, 'sap_aug_a': 0.5, 'sap_aug_s': 10, 'augment_cutmix': True, 'augment_specaug': True, 'da_prob_min': 0.1, 'da_prob_max': 0.5, 'cutmix_min': 0.075, 'cutmix_max': 0.25, 'specaug_freq_min': 0.1, 'specaug_freq_max': 0.3, 'specaug_time_min': 0.1, 'specaug_time_max': 0.3, 'specaug_freq_mask_num': 4, 'specaug_time_mask_num': 4}                                                           INFO:deepspectrumlite.cli.train:Load data pipeline ...  
INFO:deepspectrumlite.cli.train:All data pipelines have been successfully loaded.                                                                                                                                                                                                         INFO:deepspectrumlite.cli.train:Caching in memory is: True                                                                                                                                                                                                                                INFO:deepspectrumlite.cli.train:Instantiating model ...                                                                                                                                                                                                                                   INFO:deepspectrumlite.cli.train:Running model ...                                                                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:Create Keras model                                                                                                                                                                                                                               INFO:deepspectrumlite.lib.model.ai_model:Model: "resnet50"                                                                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:Layer (type)                 Output Shape              Param #                                                                                                                                                                                   INFO:deepspectrumlite.lib.model.ai_model:=================================================================                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:input_1 (InputLayer)         [(None, 224, 224, 3)]     0                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:resnet50 (Functional)        (None, None, None, 2048)  23587712                                                                                                                                                                                  INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:global_average_pooling2d (Gl (None, 2048)              0                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:flatten (Flatten)            (None, 2048)              0                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:dense (Dense)                (None, 512)               1049088                                                                                                                                                                                   INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:dropout (Dropout)            (None, 512)               0                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:dense_1 (Dense)              (None, 1)                 513                                                                                                                                                                                       INFO:deepspectrumlite.lib.model.ai_model:=================================================================                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:Total params: 24,637,313                                                                                                                                                                                                                         INFO:deepspectrumlite.lib.model.ai_model:Trainable params: 1,049,601                                                                                                                                                                                                                      INFO:deepspectrumlite.lib.model.ai_model:Non-trainable params: 23,587,712                                                                                                                                                                                                                 INFO:deepspectrumlite.lib.model.ai_model:_________________________________________________________________                                                                                                                                                                                INFO:deepspectrumlite.lib.model.ai_model:Starting model training ...                                                                                                                                                                                                                      2021-08-23 13:50:49.997666: I tensorflow/core/profiler/lib/profiler_session.cc:136] Profiler session initializing.                                                                                                                                                                        2021-08-23 13:50:49.997832: I tensorflow/core/profiler/lib/profiler_session.cc:155] Profiler session started.                                                                                                                                                                             2021-08-23 13:50:49.998198: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1365] Profiler found 1 GPUs                                                                                                                                                                           2021-08-23 13:50:49.999942: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cupti64_110.dll                                                                                                                                           2021-08-23 13:50:50.099069: I tensorflow/core/profiler/lib/profiler_session.cc:172] Profiler session tear down.                                                                                                                                                                           2021-08-23 13:50:50.099203: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1487] CUPTI activity buffer flushed                                                                                                                                                                   2021-08-23 13:50:59.536468: I tensorflow/stream_executor/platform/default/dso_loader.cc:49] Successfully opened dynamic library cudnn64_8.dll                                                                                                                                             2021-08-23 13:51:00.252830: I tensorflow/core/platform/windows/subprocess.cc:308] SubProcess ended with return code: 0                                                                                                                                                                                                                                                                                                                                                                                                                                                              2021-08-23 13:51:00.288410: I tensorflow/core/platform/windows/subprocess.cc:308] SubProcess ended with return code: 0                                                                                                                                                                                                                                                                                                                                                                                                                                                              2021-08-23 13:51:01.318948: I tensorflow/core/profiler/lib/profiler_session.cc:136] Profiler session initializing.                                                                                                                                                                        2021-08-23 13:51:01.319037: I tensorflow/core/profiler/lib/profiler_session.cc:155] Profiler session started.                                                                                                                                                                      2021-08-23 13:51:01.676758: I tensorflow/core/profiler/lib/profiler_session.cc:71] Profiler session collecting data.                                                                         2021-08-23 13:51:01.677669: I tensorflow/core/profiler/internal/gpu/cupti_tracer.cc:1487] CUPTI activity buffer flushed
